Monika After Story/game/script-fun-facts.rpy

```diff
@@ -604,3 +604,81 @@ label mas_fun_fact_getting_older:
 
     m 1hubfa "Aha, I thought so!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_fun_facts_database,
+            eventlabel="mas_fun_fact_smile_contagious",
+        ),
+        code="FFF"
+    )
+
+label mas_fun_fact_smile_contagious:
+    m 3eub "Here's one!"
+    m 1esa "Humans have a strong instinct for facial mimicry.{w=0.3} We mirror the expressions of people we see as it helps us empathize with how they're feeling."
+    m 1eua "When you smile, others smile. When you frown, others frown..."
+    m 3eub "In other words, smiling is contagious!"
+    m 1eua "Let's test it out. I'll smile at you, okay?"
+    m 6eua "{w=0.3}{nw}"
+    extend 6hua "{w=0.4}.{w=0.4}.{w=0.4}.{w=0.4}{nw}"
+    extend 6hubla "Is it working yet?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "...Is it working yet?{fast}"
+        "Yes.":
+            m 6hublb "Glad to hear it!{w=0.3} I'll make sure to keep smiling at you from now on then aha!~"
+            call mas_fun_facts_end
+            return
+        "No.":
+            m 6hubsa "{w=0.4}.{w=0.4}.{w=0.4}.{w=0.4}{nw}"
+    m 6hubsa "...How about now?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "...How about now?{fast}"
+        "Yes.":
+            m 6hubsb "Great!{w=0.3} I'll make sure to keep smiling at you from now on then aha!~"
+            call mas_fun_facts_end
+            return
+        "No.":
+            m 6hubsb "Ahaha! you're making me blush, [player]!~"
+    m 6hubfa "{w=0.4}.{w=0.4}.{w=0.4}.{w=0.4}Now?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "...Now?{fast}"
+        "Yes.":
+            m 6eubfb "Finally!{w=0.3} Looks like I'm just gonna have to start smiling at you even harder from now on aha!~"
+            call mas_fun_facts_end
+            return
+        "No.":
+            m 6hubfb "Stooooop it, [player]!~. I can't hold this smile any longer hahaha!"
+            m 1eubla "I guess it doesn't work through a computer screen..." 
+            m 1rud "Or maybe.{w=0.2}.{w=0.2}.{w=0.2}{nw}" 
+            extend 1ttu "I'm just gonna have to start smiling at you {i}reeaal{/i} hard from now on,{w=0.2} ehehe!~"
+    #Call the end
+    call mas_fun_facts_end
+    return
```

Comment:
So this entire fun fact here, is:
1. Fairly long-winded for quick topics (which fun facts are intended to be)
2. Pretty much already covered in depth in `monika_social_contagion`

This one can be removed